### ISSUE TITLE
Can I write the translation files in YAML?

### ISSUE THREAD

**Issue by rchrdnsh** [Marr11317/s20n#1]
Less syntax, can teach non-tech people to write it, can plug-in in to Netlify CMS, etc...would be nice...would something like that be possible?

**Comment by ecstrema** [comment]
Yes you can. See the [Loaders docs](https://marr11317.github.io/s20n/api/loaders).

**Comment by ecstrema** [comment]
See this REPL for an example:
https://svelte.dev/repl/cdb767b8be0041b8b23b2e6d8e4ecc5b?version=3.37.0

**Comment by ecstrema** [comment]
[This commit](https://github.com/Marr11317/s20n/commit/a8fb854bff85cf1d14b72f470e82fcb6e4163845) makes it easier to find the Loaders docs. Does that fix the issue?
